### ISSUE TITLE
strictTaskbodyConstraintMod.xsd refers to taskMod.xsd unversioned

### ISSUE THREAD

**Issue by nfekete** [dita-ot/dita-ot#2874]
## Expected Behavior
The reference to `taskMod.xsd` should be versioned as with all other references in the schema files.

## Actual Behavior
It references `taskMod.xsd` without version qualifier and it causes problems when running transformations that are based on versioned schema references with version tied to 1.2
The following command shows pretty well how `strictTaskbodyConstraintMod.xsd` is different from all other files:
```
find src/main/plugins/org.oasis-open.dita.v1_2/ -name *.xsd | xargs grep "redefine schemaLocation"
```

## Possible Solution
change schema reference from
```
<xs:redefine schemaLocation="urn:oasis:names:tc:dita:xsd:taskMod.xsd">
```
to
```
<xs:redefine schemaLocation="urn:oasis:names:tc:dita:xsd:taskMod.xsd:1.2">
```

## Steps to Reproduce
Run a transformation on a map that contains a task with `xsi:noNamespaceSchemaLocation="urn:oasis:names:tc:dita:xsd:task.xsd:1.2"`.

## Copy of the error message, log file or stack trace
The relevant log output line is:
```
src-resolve: Cannot resolve the name 'tasktroubleshooting' to a(n) 'group' component.
```

## Environment

* DITA-OT version: 2.5.4, possibly all others that support both DITA 1.2 and 1.3
* Operating system and version: independent of
* How did you run DITA-OT? `dita` command
* Transformation type: custom on top of PDF2

**Comment by nfekete** [comment]
Also this command:
```
$ find src/main/plugins/org.oasis-open.dita.v1_2/ -name *.xsd | xargs grep "<xs:\(import\|include\|redefine\)" | grep -v "1\.2"
src/main/plugins/org.oasis-open.dita.v1_2/schema/technicalContent/xsd/ditabase.xsd:  <xs:include schemaLocation="urn:oasis:names:tc:dita:xsd:ditabaseMod.xsd"/>
src/main/plugins/org.oasis-open.dita.v1_2/schema/technicalContent/xsd/ditabaseMod.xsd:<xs:import namespace="http://www.w3.org/XML/1998/namespace"  />
src/main/plugins/org.oasis-open.dita.v1_2/schema/technicalContent/xsd/strictTaskbodyConstraintMod.xsd:  <xs:redefine schemaLocation="urn:oasis:names:tc:dita:xsd:taskMod.xsd">
```

I'm not sure about it, but I think the missing version qualifier from
```
ditabase.xsd:  <xs:include schemaLocation="urn:oasis:names:tc:dita:xsd:ditabaseMod.xsd"/>
```
might also cause problems. Didn't run into it though.

**Comment by easirois** [comment]
The 1.2 was/is an SVN repository. I can bring it up at the next TC meeting.  Not sure what’s the state of the repository or the tools to build the package way back when.

**Comment by robander** [comment]
OASIS never managed to release an errata for 1.2, and given the work involved, I don't see an official one happening. The SVN repository still exists though so could be fixed there. 

Either way, this does appear to be a bug and I think is fixable in the version that ships with DITA-OT.

**Comment by robander** [comment]
Fixed with #2878

**Comment by nfekete** [comment]
Thanks for the fix! Will this be released in the 2.5 line too?

**Comment by robander** [comment]
I still think at some point we'll have another 2.5 fix, but we don't have one scheduled and my focus recetnly has been on the 3.0.2 patch. If/when we have a 2.5 fix I plan to go back through the 3.0.x hotfixes and port back any of the easy ones (this is definitely one of the easy ones).
